examples/iam/iam_api_key.md

```diff
@@ -1,40 +0,0 @@
-### Resource Creation
-
-```hcl
-resource "intersight_iam_api_key" "iam_api_key1" {
```

Comment:
This doesn't need the AppRegistration. This should still work. Right?

Reply:
iam.ApiKey cannot be created using SDK, I mean you cannot create APIKey using another APikey. so it throws same error like iam.ApprRegistration 

{"code":"InternalServerError","message":"Cannot create an Api Key. Creation of Api Key is only allowed via web sessions.","messageId":"iam_cannot_create_api_keys_without_web_session","traceId":"zwfk508OYnx_m1SdDLyAtMPTUtOHoFtDPKYoNnX1fU5drh8SiVVbjQ=="}